docs/_include/mooring.rst

```diff
@@ -92,17 +94,27 @@ The lookup table dataset should contain one structure that contains fields for e
 MoorDyn
 ^^^^^^^
 
-When the MoorDyn block is used, the user needs to initiate the mooring class by 
+When a MoorDyn block is used, the user first needs to initiate the mooring class by 
 setting :code:`mooring = mooringClass('mooring name')` in the WEC-Sim input 
-file (wecSimInputFile.m), followed by the number of mooring lines that are 
-defined in MoorDyn (``mooring(1).moorDynLines = <Number of mooring lines>``) 
-
-A mooring folder that includes a moorDyn input file (``lines.txt``) is required 
-in the simulation folder. 
+file (wecSimInputFile.m), followed by setting :code:`mooring(i).moorDyn = 1` to 
+initialize a MoorDyn connection. Each MoorDyn connection can consist of multiple 
+lines and each line may have multiple nodes. The number of MoorDyn lines and nodes in 
+each line should be defined as (``mooring(i).moorDynLines = <Number of mooring lines>``) 
+and (``mooring(i).moorDynNodes(iLine) = <Number of mooring nodes in line>``), 
+respectively. 
+
+A mooring folder that includes a MoorDyn input file (``lines.txt``) is required 
+in the simulation folder. The body and corresponding mooring attachment points are 
+defined in the MoorDyn input file. MoorDyn handles the kinematic transform to 
+convert the forces from the attachment points to the 6 degree of freedom force 
+acting on the body's center of gravity. Thus, the mooring location should generally 
```

Comment:
@jtgrasb If you are referring to a MD body here, you should note that the forces act on the body's location rather than center of gravity. The body is a bit like a point in space which when things are attached the forces and motions are controlled rigidly through that point

Reply:
Thanks for clarifying. If I'm understanding correctly, you're saying that the forces are converted to the Body location specified in lines.txt which is not necessarily the CG? WEC-Sim applies the forces to the body at the center of gravity, so the way I understand it the location specified in lines.txt should be consistent with the initial location of the center of gravity.

Reply:
You are correct. If you want the returned forces to be the forces on the CG, then the MD body needs to be initialized at the location of the CG of the WECSim Body.